hugegraph-dist/src/main/java/com/baidu/hugegraph/dist/HugeGraphServer.java

```diff
@@ -122,5 +122,11 @@ public static void main(String[] args) throws Exception {
             server.stop();
             LOG.info("HugeGraphServer stopped");
         }, "hugegraph-server-shutdown"));
+        /*
+         * HugeFactory shutdown hook be invoked before server stop, so that
+         * eventHub execute notify event on a shutdown executor, so here
```

Comment:
causes event-hub can't execute notification events for another shutdown executor such as gremling-stop-shutdown